packages/flutter_driver/lib/src/driver.dart

```diff
@@ -395,6 +396,29 @@ class FlutterDriver {
     return stopTracingAndDownloadTimeline();
   }
 
+  /// [action] will be executed with the frame sync mechanism disabled.
+  ///
+  /// By default, flutter_driver waits until there is no pending frame scheduled
+  /// in the app under test before executing an action. This mechanism is called
+  /// "frame sync". It greatly reduces flakiness because flutter_driver will not
```

Comment:
s/flutter_driver/Flutter Driver/g. That's how we refer to it in the docs here: https://flutter.io/testing/#integration-testing

(applies here and elsewhere in the PR)